### PR TITLE
fix: nested modals scrolling issue

### LINKED ISSUE #10925: Closing Nested Modals Disables Page Scrolling
### Package

filament/filament

### Package Version

v3.2.6

### Laravel Version

v10.41.0

### Livewire Version

v3.0.0.0

### PHP Version

PHP 8.2.11

### Problem description

When opening a nested modal, scroll lock is lost on the backdrop, allowing you to scroll the backdrop. After closing the nested modal (using the cancel button, x icon, or clicking off the modal) scroll lock is resumed on the backdrop. After closing the parent (or first) modal, page scrolling becomes disabled until refreshing the page or navigating away.

There are no console errors and this problem seems to occur on all browsers when using `npm run dev` or `npm run build` with `vite`.

### Expected behavior

When opening a nested modal (or slide-over), scroll lock on the backdrop should not be lost. After closing all modals, you should be able to resume scrolling the page without having to refresh or navigate away.

### Steps to reproduce

1. Install the reproduction repository locally
2. Run `composer install`
3. Run the migrations and seeders `php artisan migrate --seed`
4. Create a filament user `php artisan make:filament-user`
5. Run `npm run build`
6. Login using your credentials `/admin/login`
7. Navigate to the orders table and set the number of visible order to "all" so that the page scrolls
8. Click "New order"
9. Click "+" suffix action on either the product or customer input
10. Observe the backdrop scrolls
11. Close all modals
12. Observe the page won't scroll
13. Observer no console errors
14. Refresh the page and observe the page is scrollable again

### Reproduction repository

https://github.com/nkeena/filament-modal-bug

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by wychoong**
## Description

fixes #10925 

using $nextTick for `open` action to make sure a modal is open only after another is closed

## Visual changes

before

https://github.com/user-attachments/assets/76cd84b3-b0e2-4135-9ecf-3af727430f5a

after

https://github.com/user-attachments/assets/c7711669-822c-421d-9a48-4f1fb793fdd8

## Functional changes

- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

### CODE REVIEW

**Comment by wychoong** [comment]
if this fix is accepted, the remnant of previous attempt/rollback need to cleanup

**Comment by danharrin** [comment]
Perfect, thanks! Can you please clean up the old fix? Then I will merge.

File: packages/tables/resources/views/index.blade.php
Comment (danharrin) [inline packages/tables/resources/views/index.blade.php:1]: Hey, please fix this

Comment (wychoong) [reply packages/tables/resources/views/index.blade.php:1]: bad bot

**Pull request merged**